ui/src/components/Feedback/index.tsx

```diff
@@ -0,0 +1,138 @@
+import classNames from 'classnames/bind';
+import { Formik, FormikHelpers, FormikProps } from 'formik';
+import * as Yup from 'yup';
+
+import TextAreaField from 'src/components/Form/TextAreaField';
+import TextField from 'src/components/Form/TextField';
+import config from 'src/config';
+import { useFlash } from 'src/duck/flashMessages';
+import style from './style.module.scss';
+
+const c = classNames.bind(style);
+
+const FeedbackForm: React.FC<FormikProps<FeedbackFormValues>> = (p) => {
+  return (
+    <div className={c('container')}>
+      <form className={c('form')} onSubmit={p.handleSubmit}>
+        <div className={c('title-container')}>
+          <h2>Feedback</h2>
+          <span className={c('sub-title')}>
+            Send your complains, suggestions, or feature requests
+          </span>
+        </div>
+
+        <div className={c('email-container', 'field-container')}>
+          <span className={c('field-title')}>Email</span>
+          <TextField placeholder="Enter email" type="email" name="email" />
+        </div>
+
+        <div className={c('first-name', 'field-container')}>
+          <span className={c('field-title')}>First Name</span>
+          <TextField placeholder="Enter first name" type="name" name="first_name" />
+        </div>
+
+        <div className={c('last-name', 'field-container')}>
+          <span className={c('field-title')}>Last Name</span>
+          <TextField placeholder="Enter last name" type="name" name="last_name" />
+        </div>
+
+        <div className={c('description-container', 'field-container')}>
+          <span className={c('field-title')}>Description</span>
+          <TextAreaField name="description" placeholder="Enter description" rows={6} />
+        </div>
+
+        <button className={c('save-button')} type="submit" disabled={p.isSubmitting}>
+          <div className={c({ 'saving-in': p.isSubmitting })}>
+            {p.isSubmitting ? 'Sending' : 'Send'}
+            <span />
+          </div>
+        </button>
+      </form>
+    </div>
+  );
+};
+
+// ---
+
+export interface FeedbackFormValues {
+  description: string;
+  email: string;
+  first_name: string;
+  last_name: string;
+}
+
+const validationSchema = Yup.object().shape({
+  description: Yup.string()
+    .min(2, 'Must be 2 characters or more')
+    .max(50, 'Must be 50 characters or less')
```

Comment:
Description should have a bigger limit. 50 characters are too little for a feedback description. Let's make it 500.